Elasticity.C

```diff
@@ -1035,12 +1035,6 @@ bool ElasticityNorm::finalizeElement (LocalIntegral& elmInt)
 }
 
 
-void ElasticityNorm::addBoundaryTerms (Vectors& gNorm, double energy) const
```

Comment:
yes, we can always add it back later. i think using virtual only when it is actually overridden is good for code readability.